src/networks.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
     dustThreshold: 0 // https://github.com/dogecoin/dogecoin/blob/v1.7.1/src/core.h#L155-L160
   },
   dash: {
-    messagePrefix: '\x19DarkCoin Signed Message:\n',
+    messagePrefix: '\x19Dash Signed Message:\n',
```

Comment:
We can't change prefix that easily, we should start with https://github.com/dashpay/dash/blob/master/src/main.cpp#L90 first but that would require people to make changes in many dash-related software and must be prepared carefully.